library/src/main/java/com/cube/storm/ui/view/holder/list/ToggleableListItemViewHolder.java

```diff
@@ -52,7 +54,22 @@ public ToggleableListItemViewHolder(View view)
 
 	@Override public void populateView(final ToggleableListItem model)
 	{
-		title.populate(model.getTitle());
+		//ARCFA-229 Toggleable items announce expanded state.
+		if (model.getTitle() != null)
```

Comment:
This modification will either need to go into ARCFA codebase or the English "Description collapsed" text will need to be overridable through configuration (as not all Storm integrations are English).